client/my-sites/email/email-non-owner-message/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { SiteDetails } from '@automattic/data-stores';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { stringify } from 'qs';
+import { useEmailOwnerUserName } from 'calypso/components/data/query-email-owner-username';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { emailManagementEdit } from 'calypso/my-sites/email/paths';
+
+import './style.scss';
+
+type EmailNonOwnerMessageProps = {
+	domainName: string;
+	selectedSite?: SiteDetails | null;
+};
+
+const buildQueryString = ( parameters = {} ) =>
+	parameters ? stringify( parameters, { addQueryPrefix: true, skipNulls: true } ) : '';
+
+export const EmailNonOwnerMessage = ( props: EmailNonOwnerMessageProps ) => {
+	const { domainName, selectedSite } = props;
+
+	const translate = useTranslate();
+
+	const ownerUserName = useEmailOwnerUserName( selectedSite, domainName );
+
+	const buildLoginUrl = () => {
+		const redirectUrlParameter = emailManagementEdit(
+			selectedSite?.slug ?? '',
+			domainName,
+			'manage',
+			null,
+			'login-redirect'
+		);
+
+		return `/log-in/${ buildQueryString( {
+			email_address: ownerUserName,
+			redirect_to: redirectUrlParameter,
+		} ) }`;
+	};
+
+	const loginUrl = buildLoginUrl();
+
+	const onClickLink = ( eventType: 'login' | 'support' ) => {
+		const properties = {
+			action: eventType,
```

Comment:
```suggestion
			action: eventType,
			source: 'email-management',
```

Let's track the source of the click as well!